CHANGES.rst

```diff
@@ -293,7 +294,7 @@ Internal Changes
 - Add ``<ind-menu>`` custom element for managing drop-down menus (:issue:`5896`, :pr:`5897`,
   thanks :user:`foxbunny`)
 - Allow plugins to add extra fields to the room booking form (:pr:`6126`, thanks :user:`VojtechPetru`)
-
+- Add ``<ind-datepicker>`` custom element (:pr:`6371`, thanks :user`foxbunny`)
```

Comment:
```suggestion
- Add ``<ind-datepicker>`` custom element (:pr:`6371`, thanks :user:`foxbunny`)

```

Fixing the missing `:` in the rst user reference, and preserving the whitespace as-is (we always two blank lines at the end of a version's changelog entries).